Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -743,6 +743,10 @@ bool MainWindow::RequestStop()
     if (Core::GetState() == Core::State::Paused)
       Core::SetState(Core::State::Running);
 
+    // Tell NetPlay about the power event
+    if (NetPlay::IsNetPlayRunning())
+      NetPlay::SendPowerButtonEvent();
```

Comment:
Wouldn't it be better to put this in `TriggerSTMPowerEvent`? (In case we want to have safe shutdown plus netplay on Android or something)

Reply:
That would cause an infinite loop over the network, since that function is called from NetPlay when the packet is received.